src/components/data-exploration/cell-sets-tool/CellSetsTool.jsx

```diff
@@ -43,9 +43,8 @@ const CellSetsTool = (props) => {
   const [activeTab, setActiveTab] = useState('cellSets');
 
   const {
-    loading, error, properties, hierarchy, selected, hidden,
+    loading, error, properties, hierarchy, selected: allSelected, hidden,
```

Comment:
Why is the name change? If we were to change it I think I would go for something even more declarative like `selectedCellSets`

Reply:
I changed it to allSelected because that gets the selected cells from both tabs, after that I pass it to the hierarchical trees with the currently selected data.